hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -121,15 +133,27 @@ public Builder enableTracing(boolean enableTracing) {
       return this;
     }
 
+    public Builder setIndexToTerm(Function<Long, Long> termGet) {
+      this.indexToTerm = termGet;
+      return this;
+    }
+
     public OzoneManagerDoubleBuffer build() {
+      if (isRatisEnabled) {
+        Preconditions.checkNotNull(indexToTerm, "When ratis is enabled, " +
+                "OzoneManagerRatisSnapshot should not be null");
```

Comment:
Checking indexToTerm not null instead of omRatisSnapshot

Reply:
Done